runtime/appruntime/apisdk/app/app.go

```diff
@@ -41,11 +41,19 @@ func (app *App) Run() error {
 
 	app.Start()
 
+	// Begin serving requests.
+	serveCh := make(chan error, 1)
+	go func() {
+		serveCh <- app.api.Serve(ln)
+	}()
```

Comment:
Why this change? Seems like you want to initialize the services before you begin serving

Reply:
So this is due to the test app, it has an `init` function which calls an API, with external calls this fails because the service isn't actually running yet - which then causes it's service to fail to start up